Project.toml

```diff
@@ -1,19 +1,21 @@
 name = "GNSSSignals"
 uuid = "52c80523-2a4e-5c38-8979-05588f836870"
 authors = ["Soeren Zorn <soeren.zorn@nav.rwth-aachen.de>"]
-version = "0.15.2"
+version = "0.16.0"
```

Comment:
Is there any breaking change here, that is exported? Otherwise we should stick with v0.15.3
There is one breaking change I can see right now: `get_code_unsafe` is not exported anymore. Could we export that again so we do not have a breaking change?

Reply:
Yes, unexporting `get_code_unsafe` should be the only breaking change. I did so to avoid problems with external phase calculations requesting only integer samples. However, as the same problem occurs with `get_code`, we can also export it again.